js_modules/dagster-ui/packages/ui-components/src/components/UnstyledButton.tsx

```diff
@@ -33,4 +34,13 @@ export const UnstyledButton = styled.button<Props>`
     cursor: default;
     opacity: 0.6;
   }
+
+  ${({$outlineOnHover}) =>
+    $outlineOnHover
+      ? css`
+          &:hover {
+            outline: 1px solid ${Colors.keylineDefault()};
+          }
+        `
+      : null}
```

Comment:
I guess this is okay, but it seems like a bit of a one-off and kind of contrary to the otherwise style-less intent of the button. How would you feel about applying this as a classname at the callsite, or something similar?

Also, re: `outline`, is this intended to be different from applying a box-shadow like other bordered buttons?

Reply:
Hmm, yeah I could switch to a box-shadow. The reason I added it here is because Josh has been requesting this behavior in a few spots now (an "Unstyled" button with some styling on hover) so figured I'd add it here since there are more call sites I want to add this behavior to.